src/brogue/Items.c

```diff
@@ -3321,7 +3322,7 @@ short getLineCoordinates(short listOfCoordinates[][2], const short originLoc[2],
         }
 
         // normalize the step, to move exactly one row or column at a time
-        fixpt m = max(abs(step[0]), abs(step[1]));
+        fixpt m = max(llabs(step[0]), llabs(step[1]));
```

Comment:
[other change]

Reply:
Btw other bug fixes are ok with PRs, best to keep them as separate commits and keep the branch fixed-up so they can be viewed easily